abac/src/main/java/no/nav/common/abac/AbacHttpClient.java

```diff
@@ -19,13 +21,23 @@
 
     private final OkHttpClient client;
 
+    @Deprecated
     public AbacHttpClient(String abacUrl, String srvUsername, String srvPassword) {
         this.abacUrl = abacUrl;
         this.srvUsername = srvUsername;
         this.srvPassword = srvPassword;
         this.client = new OkHttpClient();
     }
 
+    public AbacHttpClient(String abacUrl, String srvUsername, String srvPassword, MeterRegistry meterRegistry) {
```

Comment:
Alternativt så kunne vi gitt mulighet for å sende inn OkHttpClient i konstruktøren, da blir det mer fleksibelt.
Må da lage en hjelpefunksjon ved siden av som bygger en OkHttpClient med metrikker

Reply:
Kunne kanskje ha denne og en til med OkHttpClient